demo/conjur-client-files/create_host_token.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+cd /conjurinc/ansible
+rm -rf output
```

Comment:
`rm -rf`ing all of `output/`, is that too aggressive? Do/will other scripts use `output/` too?